src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -465,4 +471,29 @@ public function setResourcesPath($resourcesPath)
     {
         $this->addResourcePath($resourcesPath);
     }
+
+
+    /**
+     * Find out whether to display environment variables from $_ENV
+     *
+     * @return array
+     */
+    public function getEnvVariables()
+    {
+        if( $this->environmentDisplay )
+            return $_ENV;
+
+        return ['PLEASE NOTE' => 'Display of environment variables has been disabled'];
```

Comment:
I'm also going to wrap these strings in double quotes, it seems that's the style you've used in the class.